x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1048,4 +1048,15 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.host_isolation.allowed',
+    first_supported_version: '8.6',
```

Comment:
Just confirming - its available in 8.6, right?

Reply:
good call, updated to say 8.6.1